classes/event/instance.php

```diff
@@ -209,7 +203,7 @@ protected function _format_return(array $calls, $return_type)
 				return json_encode($calls);
 				break;
 			case 'none':
-				return;
+				return false;
```

Comment:
I don't see or understand why this has to change? It returns `null` here by design, why should it return false now? If would break BC if people use the returned value, so if there's no compelling reason for changing this, I'd say not change it.